docs/rfcs/0007-deployment-chain.md

```diff
@@ -0,0 +1,129 @@
+- Start Date: 2021-11-03
+- Target Version: 0.22.0
+
+# Summary
+This RFC introduces a new way to enable users to deploy their complex system (or applications) in a more flexible way by manipulating the currently supporting application's deployment.
```

Comment:
```suggestion
This RFC introduces a new way to enable users to connect multiple applications to a complex deployment pipeline. It can bring to users the ability to roll out to multiple clusters gradually or promote across environments.
```